siliconcompiler/scheduler/slurm.py

```diff
@@ -259,6 +259,13 @@ def run(self):
         if defer_time:
             schedule_cmd.extend(['--begin', defer_time])
 
+        # Forward additional options
+        options = self.project.get('option', 'scheduler', 'options',
+                                      step=self.step, index=self.index)
```

Comment:
I think this should be accessed via:
```
self.project.option.scheduler.get_options(step=...)
```

Reply:
Make sense. Still getting the hang of it.